lib/posix/posix_utils.nim

```diff
@@ -99,3 +99,14 @@ proc mkdtemp*(prefix: string): string =
     raise newException(OSError, $strerror(errno))
   return $tmpl
 
+
+proc getDiskUsage*(path: string): tuple[total: uint64, used: uint64, free: uint64] {.inline.} =
+  ## Convenience proc for ``statvfs`` to get disk usage statistics in bytes.
+  ## This proc is available for 64 Bit architectures and operating systems.
+  ##
+  ## .. code-block:: nim
+  ##   echo getDiskUsage(".") ## (total: ..., used: ..., free: ...)
+  ##
+  var t = Statvfs()
+  discard statvfs(path, t)
```

Comment:
This returns an error I think, right? Would be better to check errno and throw it if the function call failed.

Reply:
@dom96 Fixed. Thanks for the Help!. I am more used to `doAssert` so I hope the `if` is Ok.